src/NzbDrone.Common/Instrumentation/Sentry/SentryTarget.cs

```diff
@@ -52,6 +54,37 @@ public SentryTarget(string dsn)
             _client.Tags.Add("culture", Thread.CurrentThread.CurrentCulture.Name);
             _client.Tags.Add("branch", BuildInfo.Branch);
             _client.Tags.Add("version", BuildInfo.Version.ToString());
+
+            _debounce = new SentryDebounce();
+        }
+
+
+        private List<string> GetFingerPrint(LogEventInfo logEvent)
+        {
+            var fingerPrint = new List<string>
+            {
+                logEvent.Level.Ordinal.ToString(),
+            };
+
+            var lineNumber = "";
+
+            if (logEvent.StackTrace != null)
+            {
+                var stackFrame = logEvent.StackTrace.GetFrame(logEvent.UserStackFrameNumber);
```

Comment:
As mentioned, get rid of the stackframe&line number stuff. Use LogEventInfo.LoggerName + LogEventInfo.Message instead.